src/Repository/CommitteeMembershipRepository.php

```diff
@@ -606,4 +609,18 @@ public function enableVoteStatusForAdherents(Committee $committee, array $adhere
             ->execute()
         ;
     }
+
+    public function committeeHasVoters(Committee $committee): bool
+    {
+        return (bool) $this->createQueryBuilder('cm')
+            ->select('COUNT(1)')
+            ->where('cm.committee = :committee AND cm.enableVote = :true')
```

Comment:
you can directly use :
```suggestion
            ->where('cm.committee = :committee AND cm.enableVote = 1')
```
no ?